README.md

```diff
@@ -70,7 +70,7 @@ When you have some local changes to the widget you'd like to test on the dashboa
 Run in the browser:
 
 ```
-nx serve desktop
+nx serve desktop -c local-stage
```

Comment:
`local-stage` is (or at least, was) the default app, no need to add the `-c` param